### ISSUE TITLE
Add control on border attribute

### ISSUE THREAD

**Issue by mrvkino** [Gruntfuggly/todo-tree#582]
Hi,

I tried to reproduce the following highlight of a tag

![image](https://user-images.githubusercontent.com/279511/143602029-2aa6d9e6-a52e-4799-9a52-5e94773c2f00.png)

But I found out it is not possible with the control given. The closest I got was:

<img width="220" alt="image" src="https://user-images.githubusercontent.com/279511/143602593-75a9656e-037d-4230-a61c-e53f8e5f67ab.png">

I would like to have the following control:
```css
"border": "1px solid #2aa198"
```

Maybe I missed something, but looking at the issue and documentation it seems border control is limited to radius.

Thanks

**Comment by mrvkino** [comment]
I was wondering why we cannot put any of the qualifiers defined in the [VSCode CSS](https://code.visualstudio.com/api/references/vscode-api#DecorationRenderOptions) instead of having each of them processed individually as done [here](https://github.com/Gruntfuggly/todo-tree/blob/5e8426c679761890de36e4a5bc83d911bc7c9fe6/src/highlights.js#L119)?

Is there a performance or discoverability idea behind that?

**Comment by Gruntfuggly** [comment]
Yes, unfortunately the styling very limited. I would also like to be able to put an outline around the highlight, but it doesn't work.

The reason the entire set of rules isn't available is that they have to be applied as a decoration and originally a lot of them were not available. It might be simpler to provide a way to specify any of them, although a lot of them will not make any sense. 

I'll add it as a feature request and see if something can be done.

**Comment by mrvkino** [comment]
In the meantime, If adding the border styling can be done in a similar fashion as the other (aka I can copy the structure), I can try to give It a shot.

I have not done much JavaScript/TypeScript development, but I can try to help.

**Comment by eversonl** [comment]
border options are now available in the API, can these be added now? I've checked and "border": "1px solid #2aa198" still doesnt work
